@benjaoming I see from the log output the `env.list` contain `KOLIBRI_WINDOWS_INSTALLER_VERSION=v1.2.1` because of that, the built are failed. Can I change the version and add it in this PR? In order to check if the PR is running with no error on Buildkite. 
```
HEAD is now at 6341241 Merge pull request #90 from learningequality/develop
--
  | make: *** No targets specified and no makefile found.  Stop.
  | make: *** [docker-windows] Error 2

export KOLIBRI_VERSION=$(cat kolibri/VERSION) && \
--
  | docker run --env-file ./docker/env.list -v $PWD/dist:/kolibridist "learningequality/kolibri-windows"
  | Cloning into 'kolibri-installer-windows'...
  | Note: checking out 'v1.2.1'.
  |  
  | You are in 'detached HEAD' state. You can look around, make experimental
  | changes and commit them, and you can discard any commits you make in this
  | state without impacting any branches by performing another checkout.
  |  
  | If you want to create a new branch to retain commits you create, you may
  | do so (now or later) by using -b with the checkout command again. Example:
  |  
  | git checkout -b <new-branch-name>
  |  
  | HEAD is now at 6341241 Merge pull request #90 from learningequality/develop
  | make: *** No targets specified and no makefile found.  Stop.
  | make: *** [docker-windows] Error 2
  | 🚨 Error: The command exited with status 2


```

--- a/docker/env.list
+++ b/docker/env.list
@@ -5,7 +5,7 @@ EXTRA_REQUIREMENTS
 KOLIBRI_VERSION
 
 # Set the default version of the Windows Installer
-KOLIBRI_WINDOWS_INSTALLER_VERSION=v1.2.1
+KOLIBRI_WINDOWS_INSTALLER_VERSION=download-python.exe
 
 # Make sure we don't record these docker runs in pingbacks
 KOLIBRI_RUN_MODE="docker"

